ES6 Module lesson: add new item in 'npm' section (#24296)
TheOdinProject/curriculum: javascript/organizing_your_javascript_code/es6_modules.md

1. Take a couple minutes to read the [About npm](https://docs.npmjs.com/getting-started/what-is-npm) page - a great introduction to npm.
2. [This tutorial](https://docs.npmjs.com/downloading-and-installing-packages-locally) teaches you how to install packages with npm.
3. [This tutorial](https://docs.npmjs.com/creating-a-package-json-file) covers the `package.json` file, which you can use to manage your project's dependencies.

> 4. Reading [this article](https://dev.to/moimikey/demystifying-devdependencies-and-dependencies-5ege) you will find out what is a development dependency and how to use them.

Reviewer: Thank you for making this pull request :rocket: 

Wording suggestion:
```suggestion
4. Read [this article on development dependencies](https://dev.to/moimikey/demystifying-devdependencies-and-dependencies-5ege) to learn what they are and how to use them.
```
